wiki/Modding/Normal_mod/id.md

```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+tags:
+  - NM
+---
+
+# Normal mod
+
+**normal mod** (*NM*) adalah sebuah [mod](/wiki/Modding) yang diberikan tanpa perlu melakukan pekerjaan apa pun kepada kreator yang memberikan mod tersebut. Normal mod biasanya dilakukan untuk mendapatkan pendapat kedua pada [beatmap](/wiki/Beatmaps) untuk membantu memperbaikinya. Banyak modder yang menggunakan istilah "NM" untuk membedakannya dari [mod for mod](/wiki/Modding/Mod_for_mod).
```

Comment:
``Normal`` should be capitalized

also `kreator` -> `mapper` as before

Reply:
whaa ok2